.travis.yml

```diff
@@ -0,0 +1,6 @@
+---
+language: python
+python:
+  - 3.5
```

Comment:
TravisCI doesn't support a generic value of "3" here, so I selected the version that is available in the Ubuntu machine image we're using for local development.